x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/_frame_layout.scss

```diff
@@ -41,8 +41,7 @@
   @include euiScrollBar;
   min-width: $lnsPanelMinWidth + $euiSize;
   overflow-x: hidden;
-  overflow-y: auto;
+  overflow-y: scroll;
```

Comment:
This will always show the scroll bar on some systems (like old Windows or some Linux distros). Not a big deal, but is it what we want?

Reply:
Yep, I'm trying to give more space to these scrolling panels and this is the best way to do it. The percentage of users on those environments are very small.